playbooks/input.yml

```diff
@@ -0,0 +1,24 @@
+---
+device_fields: "id,family,type,macAddress,managementIpAddress"
+ap_selected_field: "instanceId,displayName,internalKey,macAddress,radioDTOs"
+device_filterfield: "hostname,macAddress"
+device_filter_string: "sw2,sw1|52:54:00:0e:1c:6a"
+display_selection: 1
+
+accesspoints:
+    - macAddress: "52:54:00:0f:25:4c"
```

Comment:
We use the input variable naming convention with Snake case instead of camel case. So in this case, managementIpAddress should be management_ip_address, accesspointradiotype will be access_point_radio_type and so on. we take care of these inputs in our code in internally, whenever we have to pass it to the API.

Reply:
modified keys all to lower case. we are comparing  input file Key Value pair with the Current AP config response Key Value pair, if we change the Key this cannot compare. so that just changed keys to lower case for all including AP response

Reply:
Modified